docs/specs/change-password.md

```diff
@@ -0,0 +1,88 @@
+# Change Password
```

Comment:
- Rename this spec to `admin-api-change-password.md`
- Rename the title to `Change Password in Admin API`